readium/navigator/src/main/java/org/readium/r2/navigator/epub/EpubNavigatorViewModel.kt

```diff
@@ -178,8 +161,7 @@ internal class EpubNavigatorViewModel(
     // Serving resources
 
     val baseUrl: String =
-        baseUrl?.let { it.removeSuffix("/") + "/" }
-            ?: publication.linkWithRel("self")?.href
+        publication.linkWithRel("self")?.href
             ?: WebViewServer.publicationBaseHref
 
```

Comment:
I'm not sure about this. Why should we deal with two different cases?

Reply:
We currently support two cases:

* A remote web publication, which will have a `self` link in its manifest pointing to itself on the HTTP server. In this case, we won't use the `WebViewServer`.
* A local publication, e.g. parsed from an EPUB, which won't have any `self` link. In this case the publication is served through the `WebViewServer`.